content/doc/book/pipeline/syntax.adoc

```diff
@@ -0,0 +1,851 @@
+abort the Pipeline run.
+
+[NOTE]
+====
+A comprehensive list of available options is pending the completion of
+link:https://issues.jenkins-ci.org/browse/INFRA-1053[INFRA-1503].
+====
+
+==== parameters
+
+The `parameters` directive provides a list of parameters which a user should
+provide when triggering the Pipeline. The values for these user-specified
+parameters are made available off of the `params` object within the Pipeline,
+see the <<parameters-example>> for its specific usage.
+
+[cols="^10h,>90a",role=syntax]
+|===
+| Required
+| No
+
+| Parameters
+| _None_
+
+| Allowed
+| Only once, inside the `pipeline` block.
+|===
+
+===== Available Parameters
+
+string:: A parameter of a string type, for example: `parameters { string(name: 'DEPLOY_ENV', defaultValue: 'staging', description: '') }`
+booleanParam:: A boolean parameter, for example: `parameters { booleanParam(name: 'DEBUG_BUILD', defaultValue: true, description: '') }`
+
+
+[[parameters-example]]
+===== Example
+
+[pipeline]
+----
+// Declarative //
+pipeline {
+    agent any
+    parameters {
+        string(name: 'PERSON', defaultValue: 'Mr Jenkins', description: 'Who should I say hello to?'
+    }
+    stages {
+        stage('Example') {
+            steps {
+                echo "Hello ${params.PERSON}"
+            }
+        }
+    }
+}
+----
+
+[NOTE]
+====
+A comprehensive list of available parameters is pending the completion of
+link:https://issues.jenkins-ci.org/browse/INFRA-1053[INFRA-1503].
+====
+
+==== triggers
+
+The `triggers` directive defines the automated ways in which the Pipeline
+should be re-triggered. For Pipelines which are integrated with a source such
+as GitHub or BitBucket, `triggers` may not be necessary as webhooks-based
+integration will likely already be present. Currently the only two available
+triggers are `cron` and `pollScm`.
+
+[cols="^10h,>90a",role=syntax]
+|===
+| Required
+| No
+
+| Parameters
+| _None_
+
+| Allowed
+| Only once, inside the `pipeline` block.
+|===
+
+
+cron:: Accepts a cron-style string to define a regular interval at which the
+Pipeline should be re-triggered, for example: `triggers { cron('@daily') }`
+pollScm:: Accepts a cron-style string to define a regular interval at which
+Jenkins should check for new source changes. If new changes exist, the Pipeline
+will be re-triggered. For example: `triggers { pollScm('@daily') }`
+
+[NOTE]
+====
+The `pollScm` trigger is only available in Jenkins 2.22 or later.
+====
+
+[[triggers-example]]
+===== Example
+
+[pipeline]
+----
+// Declarative //
+pipeline {
+    agent any
+    triggers {
+        cron('@daily')
+    }
+    stages {
+        stage('Example') {
+            steps {
+                echo 'Hello World'
+            }
+        }
+    }
+}
+----
+
+==== stage
+
+The `stage` directive goes in the `stages` section and should contain one or
+more steps, an optional `agent` directive, or other stage specific directives.
```

Comment:
"and should contain one or more steps" -> "and should contain a `steps` directive"?

"stage specific" -> "stage-specific"